play-services-nearby-core-ui/src/main/kotlin/org/microg/gms/nearby/core/ui/DotChartView.kt

```diff
@@ -38,11 +38,11 @@ class DotChartView : View {
             val now = System.currentTimeMillis()
             val min = now - 14 * 24 * 60 * 60 * 1000L
             val date = Date(min)
-            val format = Settings.System.getString(context.contentResolver, Settings.System.DATE_FORMAT);
+            val format = Settings.System.getString(context.contentResolver, Settings.System.DATE_FORMAT)
```

Comment:
Why using the DATE_FORMAT at all?
It seems to be outdated and not compatible with the SimpleDateFormat (because of the missused 'm').
getMediumDateFormat is already in the context's locale.

Reply:
As described I don't know and am not happy with the output it provides either. We'll need to ask @mar-v-in who wrote this code.

Reply:
I don't remember exactly, but there was something with users being able to pick the date format manually in system settings on some Android versions which would not be reflected by locale based formatting. I guess I'd be fine with just using the getMediumDateFormat and ignore system DATE_FORMAT.

Reply:
I've updated the PR accordingly.